data/misc/amel-ogs-ilens.txt

```diff
@@ -1,9 +1,9 @@
-Group7.16	21570
-Group7.16	916
-Group7.16	2246
-Group7.16	8490
-Group7.16	314
```

Comment:
This row and the row below it were intentionally swapped to reflect the correct ordering of ziLoci corresponding to complex giLoci.